x-pack/test/cloud_security_posture_functional/page_objects/add_cis_integration_form_page.ts

```diff
@@ -245,12 +266,23 @@ export function AddCisIntegrationFormPageProvider({
 
   const getFieldValueInEditPage = async (field: string) => {
     /* Newly added/edited integration always shows up on top by default as such we can just always click the most top if we want to check for the latest one  */
-    const integrationList = await testSubjects.findAll('integrationNameLink');
-    await integrationList[0].click();
-    const fieldValue = await (await testSubjects.find(field)).getAttribute('value');
+    await navigateToEditIntegrationPage();
+    const fieldValue = await getFieldAttributeValue(field, 'value');
+    return fieldValue;
+  };
+
+  const getFieldAttributeValue = async (field: string, attribute: string) => {
+    const fieldValue = await (await testSubjects.find(field)).getAttribute(attribute);
     return fieldValue;
   };
 
+  const getElementText = async (selector: string) => {
+    const element = await testSubjects.find(selector);
+    await element.scrollIntoView();
```

Comment:
_question_: is there some issue with getting visible text when element is not in viewport? Did you try to run test without it?
Usually it is only required for actions like click/type and those actions run `scrollIntoView` by default, but maybe something changed on WebDriver API.

Reply:
I was having issues getting the visible text when navigating  between between pages. I uninstalled package after each test suite and you're correct I don't believe I need the `scrollIntoView`. Thanks!